sql/core/src/main/scala/org/apache/spark/sql/execution/LocalTableScanExec.scala

```diff
@@ -59,7 +59,9 @@ case class LocalTableScanExec(
   }
 
   override protected def stringArgs: Iterator[Any] = {
-    if (rows.isEmpty) {
+    if (rows == null) {
+      Iterator("<unknown>", output)
+    } else if (rows.isEmpty) {
```

Comment:
Is it ok to return different values in a driver and executors? How about computing this value in a driver then passing into executors?

Reply:
The value except `output` should be meaningless, otherwise assumption in related comments goes wrong and we should serialize the rows.